platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/container/ContainerNetwork.java

```diff
@@ -113,7 +115,12 @@ public List<Node> addNodes(final int count) {
         final List<RosterEntry> rosterEntries = new ArrayList<>();
         final Map<NodeId, KeysAndCerts> keysAndCerts = getKeysAndCerts(count);
 
-        for (final NodeId selfId : keysAndCerts.keySet()) {
+        // Sort the node IDs to guarantee roster entry order
+        final List<NodeId> sortedNodeIds = keysAndCerts.keySet().stream()
```

Comment:
Why is this needed? Seems fragile (and wrong) if we have to depend on the roster entry order.